src/styles/blocks/header.scss

```diff
@@ -0,0 +1,69 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+
+  &__link {
+    display: flex;
+    align-items: center;
+  }
+
+  &__nav {
+    display: flex;
+    align-items: center;
+    height: 60px;
+
+    &-list {
+      display: flex;
+      list-style: none;
+      padding: 0;
+      height: 100%;
+    }
+
+    &-item {
+      position: relative;
+      height: 100%;
+      margin: 0 10px;
+    }
+
+    &-link {
+      display: flex;
+      align-items: center;
+      text-decoration: none;
+      height: 100%;
+      font-size: $font-size-medium;
```

Comment:
Ensure that the variable `$font-size-medium` is defined in your variables file and follows the project's font-size scale. If it's not part of a predefined scale, consider adding it to maintain a consistent typographic scale.